image_op/reconstruct_traces.m

```diff
@@ -171,7 +183,11 @@ function reconstruct_traces(movie_source, ica_dir, varargin)
     pix_active = find(rec_filter>0);
     movie_portion = M(pix_active,:)';
     movie_portion(movie_portion<threshMov) = 0;
-    traces(:,cell_idx) = movie_portion * rec_filter(pix_active);  
+    trace_this = (movie_portion * rec_filter(pix_active))';  
+    % Fix baseline
```

Comment:
In fact, it would be better to factor out the baseline removal to a function of its own, in the `signal_op` directory.